src/app.ts

```diff
@@ -37,7 +37,7 @@ export class App {
         );
 
         if (!schemaValidationResult.valid) {
-            return this.failWithLoggedJsonSchemaValidationErrors(
+            this.failWithLoggedJsonSchemaValidationErrors(
```

Comment:
Doesn't this change mean that `sanityChecksPassing()` always returns true?

Reply:
Good catch.

I've changed the return type to `void` and handled that in `main.ts`.
Since the `never` methods already handle `exit` on themselves, the `exit` handling within the `main.ts` is not necessary anymore.

Reply:
Fixed here: https://github.com/laminas/laminas-ci-matrix-action/pull/83/commits/59ee75d61e8c337fed510701f81175406cae3b58